src/proj.h

```diff
@@ -313,10 +335,12 @@ double proj_xyz_dist (XYZ a, XYZ b);
 
 /* Set or read error level */
 int  proj_errno (PJ *P);
-int  proj_errno_reset (PJ *P);
 void proj_errno_set (PJ *P, int err);
-/* reduce some mental impedance in the canonical reset/restore use case */
-#define proj_errno_restore(P, err) proj_errno_set ((P), (err))
+int  proj_errno_reset (PJ *P);
+void proj_errno_restore (PJ *P, int err);
+
+char *proj_get_definition (PJ *P);
```

Comment:
Thanks for spotting this! It could result in some really nasty debugging sessions. In a recent commit, I have added the function proj_buffer_free, as (mostly) a synonym for pj_dealloc.